app/gui/integration-test/dashboard/delete.spec.ts

```diff
@@ -54,3 +55,36 @@ test('delete and restore (keyboard)', ({ page }) =>
     .driveTable.withRows(async (rows) => {
       await expect(rows).toHaveCount(1)
     }))
+
+test('clear trash', ({ page }) =>
+  mockAllAndLogin({ page })
```

Comment:
Maybe we can just setup the folders/projects using `setupAPI` here? This is faster and creating assets already tested in the other tests

Reply:
forgor! nice catch